src/multi_vector_simulator/E1_process_results.py

```diff
@@ -593,6 +593,7 @@ def convert_demand_to_dataframe(dict_values):
             {
                 dem: [
                     demands[dem][UNIT],
+                    demands[dem]["energyVector"],
```

Comment:
```suggestion
                    demands[dem][ENERGY_VECTOR],
```

Reply:
it needs to be imported at beginning of file though...

Reply:
Already imported at the beginning, so should work fine